roles/contiv_network/templates/aci_gw.j2

```diff
@@ -19,7 +19,7 @@ start)
     -e "APIC_EPG_BRIDGE_DOMAIN={{ apic_epg_bridge_domain }}" \
     -e "APIC_CONTRACTS_UNRESTRICTED_MODE={{ apic_contracts_unrestricted_mode }}" \
     --name=contiv-aci-gw \
-    contiv/aci-gw
+    {{ aci_gw_version }}
```

Comment:
> one of the marketing person just used his own image as he was using different ACI version. we had to log in to his setup and make sure that he fetches container

that is exactly my point. By allowing folks to specify an image this is bound to happen and we just increase the chances of failures.

Anyways if you would still like to give this flexibility you use two variables instead as well, something like `{{ aci_gw_image }}: {{ aci_gw_version }}` and set reasonable defaults for both in the defaults file that I mentioned above